homeassistant/components/syncthing/sensor.py

```diff
@@ -269,3 +287,211 @@ def _filter_state(self, state):
         state["label"] = self._folder_label
 
         return state
+
+
+class DeviceSensor(SensorEntity):
```

Comment:
This sounds incorrect. This basically squeezes a device into an entity with a bunch of attributes; While Home Assistant has a device construct.

Shouldn't this be a device in Home Assistant with multiple entities instead?